pkg/util/helperfunctions.go

```diff
@@ -113,18 +101,17 @@ func ResourceNameWithHash(resourceName string, kind string) string {
 	return resourceName + "-" + strconv.FormatUint(hash, 16)
```

Comment:
Did we agree to use postfix/suffixed resource type instead of hash for usability here?

Reply:
Completely forgot about that, thanks for reminding me!

I believe we indeed agreed to use a postfix resource type. Will sort out :)